### ISSUE TITLE
Overlays should minimize along with Tibia

### ISSUE THREAD

**Issue by Galarzaa90** [Mytherin/Tibialyzer#46]
I was trying to set up the Summary notification to be on display all the time and it worked, but there's one drawback. If I alt-tab to a different window or I minimize Tibia, the overlay remains on top of every window, which makes it annoying when you're trying to check the wiki or something else.

This only happens with notifications, HUDs hide by simply unfocusing the Tibia window.

![image](https://cloud.githubusercontent.com/assets/12865379/14267239/5e012d5e-fa83-11e5-8cd0-f7b1dcac0f46.png)

Using 1.2.1 beta, but this was present since 1.2 beta (I skipped 1.2 release)

**Comment by Mytherin** [comment]
I personally prefer the notifications not hiding when Tibia is minimized, but I can see why you would want that option. I will add a toggle for this sometime this week.

**Comment by Galarzaa90** [comment]
Actually, I think the opposite could be useful for HUD bars too, having the option to make them remain on screen when you switch screen. Say you're quickly checking something on your browser, and you suddenly see your mana/hp decrease so you switch back to Tibia. I don't know, maybe it's too much, sometimes I'm afraid Tibialyzer is getting so many useful features that CipSoft is going to take a second look at it and not like it haha.

**Comment by kical** [comment]
True, @Galarzaa90 Is Right there would be good an option to have an overlay of our HP while Tibia minimized, Is anoying to wait for your Jaul Turn, or wait at a respawn while watching out Youtube ;(
